api/metrics-api-jaxrs/pom.xml

```diff
@@ -141,6 +141,7 @@
       <plugin>
         <artifactId>maven-war-plugin</artifactId>
         <configuration>
+          <attachClasses>true</attachClasses>
           <archive>
```

Comment:
This is no longer needed, it was needed when I started experimenting. I will remove it.